janitor/functions.py

```diff
@@ -55,6 +57,78 @@
 )
 
 
+def get_symbol(symbol: str) -> str:
+    """
+    This is a helper function to get a companies full
+    name based on the stock symbol.
+
+    Example:
+        print(get_symbol("aapl"))
+        console >> Apple Inc.
+
+    :param symbol: This is our stock symbol that we use
+        to query te api for the companies full name.
+    :return: This is the company name
+    """
+    result = requests.get(
+        "http://d.yimg.com/autoc."
+        + "finance.yahoo.com/autoc?query={}&region=1&lang=en".format(symbol)
+    ).json()
+
+    for x in result["ResultSet"]["Result"]:
+        if x["symbol"] == symbol:
+            return x["name"]
+        else:
+            return "Not found."
```

Comment:
This makes it confusing when you can either return None or "Not Found". I think if no matches are found either due to 0 results or due to not matching any of the results, there should be a unified return (I prefer returning None). This way your test cases for ASNF and ZZZZ will return the same thing since there were no matches.

Reply:
I agree with @jk3587 here. There are some advantages, as it makes the return type compatible with pandas' `df.dropna()` function.

**Action item:** Please change the return to None.